testsuite/simulation/modelica/spatialDistribution/PlugFlowAIT.mos

```diff
@@ -0,0 +1,10 @@
+
+
+loadModel(Modelica, {"3.2.3"}); getErrorString();
+loadModel(Buildings); getErrorString();
+
+simulate(Buildings.Fluid.FixedResistances.Validation.PlugFlowPipes.PlugFlowAIT); getErrorString();
+
+simulate(Buildings.Fluid.FixedResistances.Validation.PlugFlowPipes.PlugFlowULg); getErrorString();
+
+simulate(Buildings.Fluid.FixedResistances.Validation.PlugFlowPipes.TransportWaterAir); getErrorString();
```

Comment:
add an empty line?

Reply:
Ah.. I didn't want to add these. The buildings simulations are to slow for testing. We can run them in the nightly tests.